sentry-android-integration-tests/test-app-plain/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id("com.android.application")
+}
+
+android {
+    compileSdk = 32
+
+    defaultConfig {
+        applicationId = "io.sentry.java.tests.perf.appplain"
+        minSdk = 21
+        targetSdk = 32
```

Comment:
Can we use the SDK values from `Config.Android`? So we if change it in there, it reflects here.

Reply:
changed